src/code/templates/fv-egg-game.js

```diff
@@ -819,10 +812,18 @@ export default class FVEggGame extends Component {
         this.setState({ animatingGametes: null, animatingGametesInPools: [0, 0],
                         animation: "complete", isIntroComplete: false });
       }
+      if (this.props.interactionType === "select-gametes") {
```

Comment:
I threw in some logic here so that, for the first trial in a `select-gametes` challenge, we display a `Generate Gametes` button before showing any animation or gametes.